mmv1/products/apigee/EndpointAttachment.yaml

```diff
@@ -49,6 +49,7 @@ examples:
       # This is a more verbose version of the above that creates all
       # the resources needed for the acceptance test.
   - name: 'apigee_endpoint_attachment_basic_test'
+    exclude_test: true
```

Comment:
this test is now excluded from the docs AND the test, we should just delete the template `apigee_endpoint_attachment_basic_test` at this point

Reply:
Sure, just removed